state-chain/runtime/src/lib.rs

```diff
@@ -1019,6 +1019,12 @@ impl_runtime_apis! {
 				})
 				.collect()
 		}
+		fn cf_asset_balances(account_id: AccountId) -> Vec<(ChainAsset, AssetAmount)> {
+			LiquidityProvider::asset_balances(&account_id)
+				.into_iter()
+				.map(|asset_balance| (ChainAsset {chain: asset_balance.asset.into(), asset: asset_balance.asset}, asset_balance.balance))
```

Comment:
Don't think this conversion is needed, yo seem to be going from ChainAsset to ChainAsset.